content/blog/2025/04/2025-04-28-election-announcement.md

```diff
@@ -0,0 +1,97 @@
++++
+date = "2025-04-28"
+title = "Announcing the 2025 Governing Board elections"
+
+[taxonomies]
+author = ["Robin Riley"]
+category = ["Foundation", "Elections", "Governing Board"]
++++
+
+Today we kick off the second [Governing Board](https://matrix.org/foundation/governing-board/) election season for the Matrix.org Foundation! We start with a two week nomination period that opens on Saturday, May 3rd and closes on Friday, May 16th. This year’s deadlines are anchored to the UTC timezone, rather than the [AOE](https://en.wikipedia.org/wiki/Anywhere_on_Earth) approach that we used with mixed results last year.
+
+We’ll be doing our best to reach out to every constituent group to let them know they are eligible to nominate candidates for the election.
+
+**_If you are interested in nominating someone – or yourself – to be a candidate in this election, read this post in its entirety._**
+
+<!-- more -->
+
+## Election schedule
+
+Here is the timeline for this election season, which will happen in the second quarter of every year:
+
+* Announcement on Monday, April 28
+* Nominations from Saturday, May 3 to Friday, May 16
+* Campaigning from Saturday, May 17 to Friday, May 30
+* Voting from Saturday, May 31 to Friday, June 13
+* Results announced on Monday, June 16
+
+## How our elections work
+
+Our Governing Board is made up of 9 different constituency groups across 3 categories: nonprofit and community representatives, funder representatives, and foundation representatives. Each group has been allocated seats on the board, gets to nominate its own slate of candidates, and gets to vote within that slate of candidates. 
+
+For example, Individual Members get to nominate themselves, and vote on the Individual Members who are qualified candidates. The only exception is with the Managing Director (hi, that’s me) who automatically gets a seat on the Governing Board, but that’s just one seat of many.
+
+Here is the composition of seats on the Governing Board:
+
+|Community representatives|Funder representatives|Foundation representatives|
+|:---:|:---:|:---:|
+|4 Individual Members|4 Platinum Members|3 Guardians|
+|3 Ecosystem Members|3 Gold Members|2 Spec Core Team Members|
+|2 Associate Members|2 Silver Members|Managing Director|
+
+[Learn how to become a member](https://matrix.org/membership/) and [see a list of our current organizational members](https://matrix.org/support/).
+
+We will be seating candidates for half of the constituency groups in 2025, namely: Gold, Silver, Individual, and Associate Members.
+
+![A visual election timeline for the next few years: in 2024 we elect all constituencies, in 2025 we elect Gold, Silver, Individual, and Associate Members, in 2026 we elect Platinum, Ecosystem, Guardians, and Spec Core Team Members, in 2027 we elect Gold, Silver, Individual, and Associate Members, and so on.](/blog/img/2024-election-years.png)
```

Comment:
Tinyest of all nits: I wish this was an svg :D I can count the pixels :D

Reply:
(I guess it cannot be fixed here)

Reply:
xD yeah it's not great. Earmarking this one for follow up later!